packages/open-next/src/adapters/server-adapter.ts

```diff
@@ -140,12 +140,26 @@ export async function handler(
       "public, max-age=0, s-maxage=31536000, must-revalidate";
   }
 
+  //WORKAROUND: `NextServer` does not set the correct cache response headers for stale-while-revalidate.
+  // Cloudfront expect the stale-while-revalidate value to be in seconds, but nextjs sets no value at all
+  // This causes cloudfront to not cache the stale data
+  // We fix this by setting the stale-while-revalidate value to 30 days
+  if ((headers["cache-control"] as string).includes("stale-while-revalidate")) {
+    headers!["cache-control"] = (headers["cache-control"] as string).replace(
```

Comment:
Is there a possibility that `headers.cache-control` has the time on it? If so, we'll want to use regex to replace.

Reply:
Kind of, the cache-control for ISR looks like that `s-maxage=REVALIDATE_TIME, stale-while-revalidate`. The issue here is that it's not how cloudfront process SWR. For cloudfront SWR, it expects a value, and this value is the time during which cloudfront will serve stale data while trying to revalidate in the background. https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html#stale-content